docker-compose.yml

```diff
@@ -28,6 +28,19 @@ services:
   #     - codecarbon_api
   #   restart: unless-stopped
 
+  carbonserver:
+    build: ./carbonserver
+    command: uvicorn main:app --host 0.0.0.0
```

Comment:
I suggest to add - - reload to allow dynamic reloading on file changes.

Reply:
Indeed, that is more convenient for development environment, thanks!